zerver/lib/users.py

```diff
@@ -61,15 +61,20 @@ def check_full_name(full_name_raw: str) -> str:
 # name (e.g. you can get there by reactivating a deactivated bot after
 # making a new bot with the same name).  This is just a check designed
 # to make it unlikely to happen by accident.
-def check_bot_name_available(realm_id: int, full_name: str) -> None:
+def check_bot_name_available(realm_id: int, full_name: str, is_activation: bool) -> None:
```

Comment:
Would be better to make this new parameter as a mandatory kwarg. I am not sure about the variable name, but I cannot think of a better name now, so would leave it till feedback from others.

Reply:
@sahil839, I have updated the PR with the suggessted changes.